ci-operator/step-registry/cucushift/installer/check/nutanix/day2/gpu/cucushift-installer-check-nutanix-day2-gpu-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: cucushift-installer-check-nutanix-day2-gpu
```

Comment:
I think this step is only doing `check`, but also changed the cluster configurations, if that, it is not proper to save the step in cucushift/installer/check folder, and should update the whole install workflow name accordingly to reflect it.

Reply:
The purpose is to check day2 install works, what about I restore the env in check step?

Reply:
fixed, pls have a look again, thanks.